python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -535,9 +535,9 @@ def _reduce(
         Series or DataFrame
             Computed {op} of values within each group.
 
-        Notes
-        -----
-        Difference from pandas:
+        .. pandas-compat::
+            **GroupBy.{op}**
```

Comment:
```suggestion
            **{cls}.{op}**
```

I think this will remove the need to replicate the `_reduce` method just to get the class name into the docstring correctly.